src/popup/utils/constants.ts

```diff
@@ -594,3 +596,16 @@ export const ALLOWED_ICON_STATUSES = [
   'success',
   'warning',
 ] as const;
+
+export const WEB_SOCKET_CHANNELS = {
+  Transactions: 'Transactions',
+  MicroBlocks: 'MicroBlocks',
+  KeyBlocks: 'KeyBlocks',
+  Object: 'Object',
```

Comment:
This is actually not an channel name. Also `Object` is already used in the `IMiddlewareWebSocketSubscriptionMessage` `payload` prop.

Reply:
it's a channel name :D https://github.com/aeternity/ae_mdw#publishing-message-format

used to subscribe for specific address